pkg/ovsdb/dial_windows_test.go

```diff
@@ -0,0 +1,24 @@
+package ovsdb
+
+import (
+	"testing"
+
+	"github.com/Microsoft/go-winio"
+)
+
+func TestPipeDialNet(t *testing.T) {
+	network := "pipe"
```

Comment:
in the code above there is a condition for 'winpipe' but in the test for the 'pipe' I guess one is wrong.

Reply:
Sorry,  it‘s indeed typo, change "pipie" to "winpipe" in new commit.